### PR TITLE
Update high_school_resources.html

### PR DESCRIPTION
**Pull request by BrianIngmanson**
Adding another high school resouce!

### CODE REVIEW

**Comment by Lucaman99** [comment]
Feel free to add your name somewhere in the post!

**Comment by BrianIngmanson** [comment]
Updated with name and better grammar!

File: posts/high_school_resources.html [hunk 1]
```diff
@@ -145,6 +145,21 @@ <h1>Quantum Computing Resources for High School Students</h1>
                                 online</a>.</p>
                     </li>
 
+                    
+                                        <li dir="ltr" style="padding-top:9px;">
+                        <p dir="ltr"><a
+                                href="https://quantumready.thinkific.com/">Quantum for Anybody</a></p>
+                        <p dir="ltr">This course introduces the essential concepts used in quantum 
+                            computing and explains how it's different than regular computing.
```
Comment (nathanshammah) [inline posts/high_school_resources.html:153]:
```suggestion
                            computing and explains how it's different from regular computing. 
```

File: posts/high_school_resources.html [hunk 2]
```diff
@@ -145,6 +145,21 @@ <h1>Quantum Computing Resources for High School Students</h1>
...
+                                        <li dir="ltr" style="padding-top:9px;">
+                        <p dir="ltr"><a
+                                href="https://quantumready.thinkific.com/">Quantum for Anybody</a></p>
+                        <p dir="ltr">This course introduces the essential concepts used in quantum 
+                            computing and explains how it's different than regular computing. 
+                            Made by Brian Ingmanson, a former middle school teacher, you will learn 
+                            what quantum computers can (and can't) do right now and guide you 
+                            through running code on a real quantum computer. This is a true beginner level 
+                            course, so no linear algebra or tough equations will be used. That being said, 
+                            you will NOT have enough background to go get a job after this course! You'll
```
Comment (nathanshammah) [inline posts/high_school_resources.html:158]:
```suggestion
                            you will <em>not</em> have enough background to go get a job after this course! You'll 
```

**Pull request merged**